www/custom.css

```diff
@@ -209,4 +209,11 @@ a:hover {
 /* || CHANGELOG */
 #shiny-tab-tabChangelog td:first-child {
   white-space:nowrap;
+}
+
+#fileInfo.recalculating::before {
+  content: "Your file is being processed";
+}
+#fileInfo.recalculating p {
```

Comment:
this hides the "Successfull upload" or "Please upload file" texts while the new file is being processed.